mindsdb/integrations/handlers/financial_modeling_prep_handler/financial_modeling_handler.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+from typing import Dict
+from urllib.parse import urlencode
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.integrations.handlers.financial_modeling_prep_handler.financial_modeling_tables import HistoricalPriceTable
+
+from urllib.request import urlopen
+from mindsdb.utilities import log
+import json
+import requests
+_FINANCIAL_MODELING_URL = 'https://financialmodelingprep.com/api/v3/'
+
+logger = log.getLogger(__name__)
+
+class FinancialModelingHandler(APIHandler):
+
+    name = "financial_modeling"
+
+    def __init__(self, name, connection_data: dict,  **kwargs):
+        super().__init__(name)
+
+        self.api_key = None
+        self.connection_data = connection_data
+        if "api_key" not in connection_data:
+            raise Exception(
+                "FINANCIAL_MODELING engine requires an API key. Retrieve an API key from https://site.financialmodelingprep.com/developer. See financial_modeling_prep_handler/README.MD on how to include API key in query."
+            )
+        self.api_key = connection_data['api_key']
+        self.client = None
+        self.is_connected = False
+
+        historical_price_table = HistoricalPriceTable(self) 
+        self._register_table('historical_price_table', historical_price_table)
```

Comment:
Can we give this table a more meaningful name? We should definitely remove `_table` from the name. Maybe `historial_prices` or something else you think is more appropriate?